src/i18n/locales/de-DE.json

```diff
@@ -27,15 +27,15 @@
         "previewButton": "Vorschaumodus",
         "previewButton-tooltip": "Überspringt gelöschte Segmente bei der Wiedergabe des Videos. Derzeit {{status}}. Hotkey: {{hotkeyName}}",
         "previewButton-aria": "Vorschaumodus aktivieren oder deaktivieren. Hotkey: {{hotkeyName}}.",
-        "playButton-tooltip": "Play-Button",
-        "time-duration-tooltip": "Wiedergabezeit und -dauer",
+        "playButton-tooltip": "Video abspielen",
+        "current-time-tooltip": "Aktuelle Zeit",
```

Comment:
Arguably does not matter in this case, but I think it might be nicer to submit changes to existing translation keys through crowdin, just to keep to fair to the translators. (Unless of course it makes sense to include the change in the pull request because e.g. the function associated with the translated text changed).

Reply:
I submitted the change to crowdin.